### ISSUE TITLE
Initial Tensor => Tensor predictions based on frozen models

### ISSUE THREAD

**Issue by lossyrob** [azavea/geotensorflow#3]
Scala model that can be created by frozen model.

Prediction method that takes in Input tensor and produces correct Output tensor.

Unit tests that test this over a variety of models, normalizations, tasks, etc.

**Comment by yoninachmany** [comment]
Notes from 9/27:

Start with Python model that can be created from frozen Keras ResNet50 and InceptionV3 models with weights pre-trained on ImageNet.

Prediction method that takes in Input tensor and produces correct Output tensor, in Python.

Unit tests over the Keras tagging models.

Repeat in Scala.

Repeat in Scala with Raster Vision baseline tagging model from 6/22/17.
